scripts/scil_surface_apply_transform.py

```diff
@@ -8,7 +8,7 @@
 Example usage from T1 to b0 using ANTs transforms:
 > ConvertTransformFile 3 output0GenericAffine.mat vtk_transfo.txt --hm
 > scil_surface_apply_transform.py lh_white_lps.vtk affine.txt lh_white_b0.vtk\\
-    --ants_warp warp.nii.gz
+    --in_deformation warp.nii.gz --inverse
```

Comment:
Could you use vtk_transfo.txt in the example (since it is the output filename from the line above) and use the output1InverseWarp.nii.gz file